src/react-app-env.d.ts

```diff
@@ -1 +1,51 @@
 /// <reference types="react-scripts" />
+
+export interface User {
+  id: number,
+  name: string,
+  username: string,
+  email: string,
+  address: Address,
+  phone: string,
+  website: string,
+  company: Company,
```

Comment:
It is considered to be bad practice to use types or values before they are defined (according to ESLint style guides). This interface should go after interface Company.